plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1195,7 +1195,7 @@ def onNextSampleButton(self):
 
             logging.info(f"Check if file exists/shared locally: {image_file} => {local_exists}")
             if local_exists:
-                self._volumeNode = slicer.util.loadVolume(image_file)
+                self._volumeNode = slicer.util.loadVolume(image_file, {"discardOrientation": True})
```

Comment:
https://github.com/Project-MONAI/MONAILabel/issues/22

Reply:
Discarding the orientation is an ugly workaround, which might be temporarily acceptable for a specific data set, while working on the proper solution. Still, add plenty of comments explaining why it was necessary for this specific case. 

Discarding orientation may sound like discarding some uninteresting detail but it actually completely ruins your data set, because you can never recover from it. You generate a new data set that is spatially not related to the input data set anymore. It is so bad because you don't know what information you've discard, so you cannot ever reverse the operation.

Applying an explicit RAS->LPS conversion (by multiplying the IJKToRAS transform by `diag(-1,-1,1,1)`) is a much more manageable workaround. But if we talk about workarounds then a much cleaner solution would be use a loader to split the volume into 3D volumes before loading or use a different nifti loader to get the 4D numpy array and IJKToRAS matrix and create a volume sequence from it.

We could add 4D Nifti support to Slicer, but we need strong driving use cases. If we start seeing that 4D Nifti is adopted by the community then it would give motivation. But in this case it seems that somebody just dumped completely unrelated 3D images into a 4D sequence. I don't think this is a good practice. Image sequences are intended for storing tightly related 3D volumes (e.g., time sequence, vector components, etc.), but not for storing images that are acquired separately, using different imaging protocols (in this case, T1, T1CE, T2, FLAIR). You cannot even specify the metadata in the Nifti file (e.g., which frame stores which protocol), so this is definitely not an archival format. A user can temporarily merge these volume into a single file during training but this is certainly not necessary, and it should not be done if it causes more problems than the inconveniences of using 4 separate files.

Who created these 4D Nifti files?

Reply:
That said, if you say that 4D Nifti support is the most important missing feature in Slicer for MONAILabel then we can certainly take care of it. Just want to make sure we are working on things that add the most value.

Reply:
I agree with @lassoan 's assessment here.  Reading 4D nifti as a sequence or a vector volume would make sense if there are common enough use case that make the conventions are standardized and meaningful.  The current Slicer behavior here is suboptimal though.  I'm thinking we may need a module that imports nifti more like the way [RawImageGuess](https://discourse.slicer.org/t/new-extension-rawimageguess-for-loading-of-images-from-unrecognized-file-format/9219) deals with semi-structured data.